libs/block-editor/src/lib/extensions/components/bubble-change-dropdown/bubble-change-dropdown.component.ts

```diff
@@ -0,0 +1,160 @@
+                    this.editor.chain().focus().clearNodes().setParagraph().run();
+                    this.showSuggestions = false;
+                }
+            }
+        ];
+
+        const list = [
+            {
+                label: 'List Ordered',
+                icon: this.sanitizeUrl(olIcon),
+                isActive: () => this.editor.isActive('orderedList'),
+                command: () => {
+                    this.editor.chain().focus().clearNodes().toggleOrderedList().run();
+                    this.showSuggestions = false;
+                }
+            },
+            {
+                label: 'List Unordered',
+                icon: this.sanitizeUrl(ulIcon),
+                isActive: () => this.editor.isActive('bulletList'),
+                command: () => {
+                    this.editor.chain().focus().clearNodes().toggleBulletList().run();
+                    this.showSuggestions = false;
+                }
+            }
+        ];
+
+        const block = [
+            {
+                label: 'Blockquote',
+                icon: this.sanitizeUrl(quoteIcon),
+                isActive: () => this.editor.isActive('blockquote'),
+                command: () => {
+                    this.editor.chain().focus().clearNodes().toggleBlockquote().run();
+                    this.showSuggestions = false;
+                }
+            },
+            {
+                label: 'Code Block',
+                icon: this.sanitizeUrl(codeIcon),
+                isActive: () => this.editor.isActive('codeBlock'),
+                command: () => {
+                    this.editor.chain().focus().clearNodes().toggleCodeBlock().run();
+                    this.showSuggestions = false;
+                }
+            }
+        ];
+
+        this.menuItems = [...headings, ...paragraph, ...list, ...block];
+        this.editor.on('transaction', () => {
+            this.setSelectedItem();
+        });
+
+        this.editor.on('create', (editor: any) => {
+            console.log('create: ', editor);
+            this.editor.options.element.addEventListener('keydown', (event: KeyboardEvent) => {
+                const { key } = event;
+                if (this.showSuggestions) {
+                    if (key === 'Escape') {
+                        this.showSuggestions = false;
+                        return true;
+                    }
+
+                    if (key === 'Enter') {
+                        this.suggestions.execCommand();
+                        event.preventDefault();
+                        event.stopPropagation();
+                        return false;
+                    }
+
+                    if (key === 'ArrowDown' || key === 'ArrowUp') {
+                        this.suggestions.updateSelection(event);
+                        return true;
+                    }
+
+                    return false;
+                }
+            });
+        });
+    }
```

Comment:
This shouldn't live in the component, our components are dump, all this even handling should happen in the plugin.

https://github.com/dotCMS/core-web/blob/master/libs/block-editor/src/lib/plugins/floating.plugin.ts#L225